.rubocop.yml

```diff
@@ -23,7 +23,7 @@ AllCops:
     - 'dashboard/db/schema.rb'
     - 'pegasus/test/test_string.rb' # Parser does not support non-utf8 escape sequences
   DisplayCopNames: true
-  TargetRubyVersion: 2.7
+  TargetRubyVersion: 3.3
```

Comment:
I considered moving this to the "actual upgrade" PR, but `rake lint` does pass set like this, so I guess might as well have it here 🤷